### PR TITLE
Made Readme more accessible to first time contributors

### PR DESCRIPTION
**Pull request by dhruv-porwal**
Added new content

## Describe your changes 
Made Readme more accessible  to first time contributors by adding comprehensive steps to contribute 
Please include a summary of the changes and the related issue. Why is this change required? What problem does it solve? 
Added steps for convenience of  first time contributors

Fixes # (issue)
increases accessibility
# Checklist: 
- [ yes] I have performed a self-review of my code 
- [yes ] I have commented my code
- [yes ] My changes generate no new warnings 
- [yes ] Have you successfully run tests with your changes locally?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,119 @@ keytool -list -v -keystore "c:\users\your_user_name\.android\debug.keystore" -al
...
+If you think that you can add a new feature or want to fix a bug, we invite you to contribute to YADA and make this project better. To start contributing, follow the below instructions:
+
+1. Create a folder at your desire location (usually at your desktop).
+
+2. Open Git Bash Here
+
+3. Create a Git repository.
+
+   Run command `git init`
+
```
Comment (yellowHatpro) [inline README.md]: You can remove this part. Can begin with Fork and clone command

File: README.md [hunk 2]
```diff
@@ -23,6 +23,119 @@ keytool -list -v -keystore "c:\users\your_user_name\.android\debug.keystore" -al
 You can share the SHA-1 Key with your mentor, and they will add it in Firebase console.
 If you face any problem in this step, make sure to contact the mentor.
 
+
+
+
+
+# 🖱️ How to Contribute
```
Comment (yellowHatpro) [inline README.md]: Please make it ## instead of #

**Comment by yellowHatpro** [comment]
@dhruv-porwal , thanks for the work. I've left some comments, you can work on them

**Comment by dhruv-porwal** [comment]
> @dhruv-porwal , thanks for the work. I've left some comments, you can work on them

Where sir?

**Comment by dhruv-porwal** [comment]
Sir please merge it.

**Review by yellowHatpro — APPROVED** [review]

**Pull request merged**